tests/unittests/net/network_configs.py

```diff
@@ -2265,41 +3495,41 @@
     bond-downdelay 10
     bond-fail-over-mac active
     bond-master bond0
+    bond_miimon 100
```

Comment:
is this change intentional? (repeat the same question for the next several changes)

Reply:
Yes. Github unfortunately renders this very poorly. You're looking at the v2 addition, and when using v2 things get rendered alphabetically. This is a change between v1 and v2 rendering, but it results in no functional difference so it is fine.

This may better demonstrate the difference here. Notice the left has expanded the expected v1 whereas the right has expanded the expected v2:
![image](https://github.com/canonical/cloud-init/assets/153674/88352994-4385-46d5-9aa7-1c16378a5753)